plugins/modules/identity/bitwarden.py

```diff
@@ -0,0 +1,699 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2022, Jonathan Lung (@lungj) <lungj@heresjono.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+module: bitwarden
+author:
+    - Jonathan Lung (@lungj)
+requirements:
+    - C(bw) Bitwarden command line utility. See U(https://bitwarden.com/help/cli/)
+    - Organization names in Bitwarden are unique.
+    - Folders in Bitwarden have no duplicate names per organization.
+    - The name of items in folders uniquely identifies the items per organization.
+notes:
+    - Tested with C(bw) version 2022.6.2
+short_description: Manipulate items in Bitwarden
+description:
+    - M(community.general.bitwarden) wraps the C(bw) command line utility to modify Bitwarden-compatible vaults.
+    - A fatal error occurs if any of the items being searched for can not be found.
+    - Recommend using with the C(no_log) option to avoid logging the values of the secrets being retrieved.
+options:
+    cli_path:
+        type: path
+        description: Used to specify the exact path to the C(bw) command line interface
+        required: false
+        default: 'bw'
+    organization_name:
+        type: str
+        description:
+            - Name of organization to target.
+            - Leave as null for a wildcard vault or C('') for personal vaults.
+            - Mutually exclusive from organization_id.
```

Comment:
```suggestion
            - Mutually exclusive from I(organization_id).
```
See https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation